examples/react-ts/main.ts

```diff
@@ -1,4 +1,4 @@
-import type { StorybookConfig } from '@storybook/core/types';
+import type { StorybookConfig } from '@storybook/react/types';
```

Comment:
@hipstersmoothie is getting rid of all these separate entry points in https://github.com/storybookjs/storybook/pull/12462 ... i don't fully understand the rationale, but i'm wondering whether we should follow that pattern here

Reply:
I'm not sure, but wouldn't it be better if we would merge this and take care of the detail either in his PR or after that's been merged?

Reply:
CRA does not yet support import type which forces me to extract config typings in a dedicated place.
Using existing typings causes "import type" being use by storybook and react-docgen-typescript with the crashes we saw on CI (reproduced locally)